sessions/js-fetch/star-wars/components/Card/Card.css

```diff
@@ -0,0 +1,13 @@
+.card {
+	padding: 20px;
+	border-radius: 14px;
```

Comment:
Any reason why specifically 14px?

Reply:
No, no reason at all, all styling is taken from the old PR and I did not change it. Which value would you suggest?